qiskit/algorithms/minimum_eigen_solvers/vqe.py

```diff
@@ -517,7 +539,7 @@ def compute_minimum_eigenvalue(
 
         if aux_operators is not None:
             aux_values = self._eval_aux_ops(opt_params, aux_operators, expectation=expectation)
-            result.aux_operator_eigenvalues = aux_values[0]
+            result.aux_operator_eigenvalues = aux_values
```

Comment:
I am not exactly sure why this was only storing the first entry.. @woodsp-ibm do you know this?

Reply:
In looking it seems to be the weird way it was built out. Starts out with a List which it then nests, when it does an np.array aa below, and then in the above it unnested it
https://github.com/Qiskit/qiskit-terra/blob/bd537b161ce6c06e929e623f91d46fe54a61f00b/qiskit/algorithms/minimum_eigen_solvers/vqe.py#L429-L436